casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -51,9 +51,10 @@ object Genesis {
                     initial: BlockMessage,
                     startHash: StateHash,
                     runtimeManager: RuntimeManager)(implicit scheduler: Scheduler): BlockMessage = {
-    val Right(checkpoint) = runtimeManager.computeState(startHash, blessedTerms)
-    val stateHash         = ByteString.copyFrom(checkpoint.root.bytes.toArray)
-    val reductionLog      = checkpoint.log.map(EventConverter.toCasperEvent)
+    //TODO: ignore `deployCost` of blessed contracts?
```

Comment:
Yeah, the genesis block is created for free to get the network going. So we can track the cost, but it doesn't really mean anything per se.